README.Rmd

```diff
@@ -91,8 +91,8 @@ Forecasts can be easily and quickly scored using the `score()` function. `score(
 example_quantile %>%
   set_forecast_unit(c("location", "target_end_date", "target_type", "horizon", "model")) %>%
   validate() %>%
+  add_coverage() %>%
```

Comment:
if its in the default metrics list what does this actually do? Is it going to be clear to users why it isn't part of score?

Reply:
Updating the readme is on the list: https://github.com/epiforecasts/scoringutils/issues/439